samples: samples/arch/mpu/mpu_test: frdm_mcxe31b/mcxe31b: error: Error during pytest execution

**Describe the bug**
below case are error
sample.mpu.mpu_test.test_shell_harness

test is error on v4.3.0-rc1 on frdm_mcxe31b/mcxe31b
testcase path is samples/arch/mpu/mpu_test

see logs for details

**To Reproduce**
1. 
```
scripts/twister -p frdm_mcxe31b/mcxe31b  -T samples/arch/mpu/mpu_test -s samples/arch/mpu/mpu_test
```
or
```
# west twister -p frdm_mcxe31b/mcxe31b -T samples/arch/mpu/mpu_test --sub-test samples/arch/mpu/mpu_test
```
2. See error

```
Error during pytest execution
failed on setup with "Failed: Prompt not found"
```

**Expected behavior**
test pass

**Impact**


**Logs and console output**

```
dut = HardwareAdapter()

    @pytest.fixture(scope=determine_scope)
    def shell(dut: DeviceAdapter) -> Shell:
        """Return ready to use shell interface"""
        shell = Shell(dut, timeout=20.0)
        if prompt := find_in_config(Path(dut.device_config.app_build_dir) / 'zephyr' / '.config',
                                    'CONFIG_SHELL_PROMPT_UART'):
            shell.prompt = prompt
        logger.info('Wait for prompt')
        if not shell.wait_for_prompt():
>           pytest.fail('Prompt not found')
E           Failed: Prompt not found

../../zephyr/scripts/pylib/pytest-twister-harness/src/twister_harness/fixtures.py:81: Failed
    
```

```
Adapter()
@pytest.fixture(scope=determine_scope)
def shell(dut: DeviceAdapter) -> Shell:
"""Return ready to use shell interface"""
shell = Shell(dut, timeout=20.0)
if prompt := find_in_config(Path(dut.device_config.app_build_dir) / 'zephyr' / '.config',
'CONFIG_SHELL_PROMPT_UART'):
shell.prompt = prompt
logger.info('Wait for prompt')
if not shell.wait_for_prompt():
>           pytest.fail('Prompt not found')
E           Failed: Prompt not found
../../zephyr/scripts/pylib/pytest-twister-harness/src/twister_harness/fixtures.py:81: Failed
------------------------------ Captured log setup ------------------------------
DEBUG    twister_harness.device.factory:factory.py:37 Get device type "hardware"
DEBUG    twister_harness.device.hardware_adapter:hardware_adapter.py:158 Opening serial connection for /dev/serial/by-id/usb-NXP_Semiconductors_MCU-LINK_FRDM-MCXE31B__r2E4__CMSIS-DAP_V3.148_CIH4GQ4MK4RCL-if02
DEBUG    twister_harness.device.hardware_adapter:hardware_adapter.py:126 Flashing device CIH4GQ4MK4RCL
DEBUG    twister_harness.device.hardware_adapter:utils.py:37 Flashing command: /home/ubuntu/zephyrproject/.venv/bin/west flash --skip-rebuild --build-dir /home/ubuntu/nxp/frdm_mcxe31b/temp/frdm_mcxe31b_twister_test_only/twister-out/frdm_mcxe31b_mcxe31b/zephyr/samples/arch/mpu/mpu_test/sample.mpu.mpu_test --runner linkserver --probe=CIH4GQ4MK4RCL
DEBUG    twister_harness.device.hardware_adapter:hardware_adapter.py:150 Flashing finished
DEBUG    twister_harness.helpers.utils:utils.py:24 Not found key: CONFIG_SHELL_PROMPT_UART
INFO     twister_harness.fixtures:fixtures.py:79 Wait for prompt
- generated xml file: /home/ubuntu/nxp/frdm_mcxe31b/temp/frdm_mcxe31b_twister_test_only/twister-out/frdm_mcxe31b_mcxe31b/zephyr/samples/arch/mpu/mpu_test/sample.mpu.mpu_test/report.xml -
=========================== short test summary info ============================
ERROR ../../zephyr/scripts/pylib/shell-twister-harness/test_shell.py::test_shell_harness
============================== 1 error in 45.53s ==============================

```

**Environment (please complete the following information):**
 - OS: (e.g. Linux ): Linux
 - Toolchain (e.g Zephyr SDK): zephyr
 - Commit SHA or Version used: v4.3.0-rc1

--- a/boards/nxp/frdm_mcxe31b/frdm_mcxe31b.dts
+++ b/boards/nxp/frdm_mcxe31b/frdm_mcxe31b.dts
@@ -30,6 +30,7 @@
 		zephyr,flash = &program_flash;
 		zephyr,flash-controller = &flash;
 		zephyr,console = &lpuart_5;
+		zephyr,shell-uart = &lpuart_5;
 	};
 
 	leds {

